content/en/docs/concepts/security/security-checklist.md

```diff
@@ -165,11 +165,15 @@ For historical context, please note that Docker has been using
 [a default seccomp profile](https://docs.docker.com/engine/security/seccomp/)
 to only allow a restricted set of syscalls since 2016 from
 [Docker Engine 1.10](https://www.docker.com/blog/docker-engine-1-10-security/),
-but Kubernetes is still not confining workloads by default. The default seccomp
-profile can be found [in containerd](https://github.com/containerd/containerd/blob/main/contrib/seccomp/seccomp_default.go)
-as well. Fortunately, [Seccomp Default](/blog/2021/08/25/seccomp-default/), a
-new alpha feature to use a default seccomp profile for all workloads can now be
-enabled and tested.
+but Kubernetes 1.27 defaults to not confining workloads with seccomp. This means that if a workload does not explicitly 
```

Comment:
```suggestion
but Kubernetes v{{< skew currentVersion >}} defaults to _not_ confining workloads with seccomp.
This means that if a workload does not explicitly 
```

Reply:
I don't agree to this suggestion.
IMHO, we should use explicit version numbers when something started or ended.
This behavior change started in v1.27, not v1.28 or v1.29.

When we say "Kubernetes v1.27 starts to support feature Foo, while deprecating feature Bar", this version number is not supposed to be changed when v1.28 is out.
Uses like me would like to know from the text if the changes have an impact on my cluster. If we use the `skew` shortcode, the text will become something like "Kubernetes v1.28 starts to support feature Foo, while deprecating feature Bar". That is confusing and incorrect.

Reply:
It's problematic. “Kubernetes 1.27 defaults to not confining workloads with seccomp” doesn't capture what happens in Kubernetes 1.28 - we're leaving it to the reader to infer.

We can't write:
```markdown
Kubernetes versions 1.27 and later default to _not_ confining workloads with seccomp.
```
or
```markdown
Kubernetes versions starting at 1.27 default to _not_ confining workloads with seccomp.
```
because both of those accidentally a statement about the future - and they might be incorrect.

Perhaps the best we can do is to make sure that we revise this text each minor release, eg after the next release
```markdown
but Kubernetes 1.27 and 1.28 default to _not_ confining workloads with seccomp. This means that
if a workload does not explicitly 
```
---
For now, let's not change it. However @SaadOps, if you can think of wording you now prefer, please feel free to suggest that.